templates/stream.html

```diff
@@ -1,17 +1,23 @@
 {% extends "base.html" %} {% block content %}
 
-<h1>socialDISTRIBUTION</h1>
 <div class="container">
-	<a href="{% url 'posts:new' %}">New post</a>
-
+	<ul>
+		<li><a href="{% url 'posts:new' %}">New post</a></li>
+		<li><a href="Friend Requests">Friend Requests</a></li>
+	</ul>
```

Comment:
Let's make a partial for this (e.g. `_appbar_partial.html`) then include it in this file

Reply:
sure, that sounds like a good idea

Reply:
Let's also use HTML semantics
```
<style>
<!-- put this in styles.css -->
.appBar {
  display: flex;
  flex-direction: row;
</style>

<nav class="appBar">
  <ul>
    <li><a href="">linkName</a></li>
  </ul>
</nav>
```
And the links could be
1. SocialDistribution (align left)
2. spacer (span with flex grow)
3. Inbox
4. FriendRequests
5. UserProfile

My bad I should've specified this in the issue

Reply:
Done